lightspark-sdk/src/commonMain/kotlin/com/lightspark/sdk/LightsparkCoroutinesClient.kt

```diff
@@ -729,16 +729,22 @@ class LightsparkCoroutinesClient private constructor(
      *
      * @param nodeId The ID of the node to fund. Must be a REGTEST node.
      * @param amountSats The amount of funds to add to the node. Defaults to 10,000,000 SATOSHI.
+     * @param fundingAddress: L1 address owned by funded node
```

Comment:
Also maybe good to say what `null` means in this context (that it automatically creates a new funding address).